.github/workflows/generate-documentation.yml

```diff
@@ -0,0 +1,28 @@
+name: Generate documentation
+
+on:
+  workflow_dispatch:
```

Comment:
:question: I guess we will configure inputs for doc component and branch sources later? Probably when we will have not only BICI as component

Reply:
Yes, when we want generate only one sub part of our site. For now we want rebuild and redeploy all.